Rakefile

```diff
@@ -29,6 +29,21 @@ namespace :test do
   end
 end
 
+task :deps do
+  src = `curl -sL https://github.com/inspec/inspec-aws/archive/master.tar.gz | tar xO inspec-aws-master/Gemfile`
```

Comment:
This task isn't going to be usable by Windows users. Since the task is informational and not critical to using the plugin, that's probably OK.